.travis.Dockerfile

```diff
@@ -5,6 +5,8 @@ RUN apt-get -qq update && \
 
 ADD https://storage.googleapis.com/golang/go1.9.2.linux-amd64.tar.gz /tmp
 
+COPY registries.conf /etc/containers/registries.conf
```

Comment:
Hum, do we want c/image to _require_ that file to be present?  It seems reasonable to me for `sysregistriesv2` to silently use an empty configuration if the file does not exist (but still fail if it is unreadable for other reasons, e.g. permission or I/O error).

OTOH maybe the file will always be present in practice and this is not worth worrying about; @rhatdan ?

Reply:
No we should make that file optional.